Saint-Nec-Engine/Saint-Nec-Engine-Lib/src/architecture/GameObject.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include <vector>
+#include "Component.h"
+
+namespace saintNecEngine
+{
+
+	class GameObject
+	{
+		std::vector<int> position;
```

Comment:
Du coup on repart sur du vect ? Sinon quitte à vouloir une struct. pour pouvoir faire le .x / .y / .z ... autant mettre directement dans la classe ou créer une classe "Coordonnée" non ?

Reply:
I need to think of what the shader will take as an input, it will probably change